### ISSUE TITLE
Feature Request: Browser Shows When Runs Have Finished Processing

### ISSUE THREAD

**Issue by bdorney** [cms-gem-daq-project/ldqm-browser#7]
Could we add a way for the webpage browser to show when a run has finished processing?  For example in the run list could there finished runs be colored "green," processing runs be colored "orange" and pending runs be colored maybe "blue" or something.

Alternatively when viewing the run could a simple text field be shown at the top (above AMC/GTX selection) where it reads:

"Run has finished processing"
"Run is processing"
"Run is pending"

Or someting similar.

**Comment by mexanick** [comment]
Yes, we can do it. I'm adding Jared to the thread. 
There's a field in Run model, called Status (https://github.com/cms-gem-daq-project/ldqm-browser/blob/master/LightDQM/ldqm_db/models.py#L45)
This field was added for such feature. However, we need an interaction with XDAQ in order to know whether the run is ongoing, or it is stopped. 
I propose following scheme:
run.Status=0 --> run is ongoing
run.Status=1 --> run is stopped by XDAQ.
then, 
- if run.Status is 0 or 1 and there're chunks to be processed, run is processing
- if run.Status is 0 and there's no new chunks, run is pending
- if run.Status is 1 and there's no new chunks, run has finished the processing. 

Would you agree with such scheme?

**Comment by bdorney** [comment]
Yes sounds good.  Let's move forward with a test and then we can evaluate if further tweaks are needed.  Thanks for proposing this!
